src/02-data-types.js

```diff
@@ -111,7 +129,13 @@ function isEven(num) {}
  *  isOdd(11);
  *  //> true
  */
-function isOdd(num) {}
+function isOdd(num) {
+  if (num%2 === 1){
```

Comment:
Here too! We can just return `num % 2 === .` and our some spaces between our operator and values.